packages/js/data/src/utils.ts

```diff
@@ -21,6 +21,25 @@ export function getResourceName(
 	return `${ prefix }:${ identifierString }`;
 }
 
+/**
+ * Generate a resource name for order totals count.
+ *
+ * It omits query parameters from the identifier that don't affect
+ * totals values like pagination and response field filtering.
+ *
+ * @param {string} prefix Resource name prefix.
+ * @param {Object} query  Query for order totals count.
+ * @return {string} Resource name for order totals.
+ */
+export function getTotalOrderCountResourceName(
+	prefix: string,
+	query: Record< string, unknown >
+) {
+	const { _fields, page, per_page, ...totalsQuery } = query;
```

Comment:
Is there a more robust way to handle this? It seems like this is brittle, requiring updating in the future if additional query params are added that don't affect totals. Perhaps we should explicitly list what query params we do want, instead of which ones we don't want?

What query params do we want? Are there any?

Reply:
> What query params do we want? Are there any?

The problem with the params we do want is that they can change depending on what product you are querying for. For the product categories we will want the `parent`, `id`, `menu_order`, `count`, `slug`, `display`. But this might be different for just `products` or attribute terms.
`_fields`, `page`, and `per_page` are generic across all the REST api GET requests, and are currently the only items that don't affect the total of number of items being returned.

Reply:
Makes sense.